common/script/fns/statsComputed.js

```diff
@@ -0,0 +1,28 @@
+import _ from 'lodash';
+import content from '../content/index';
+import splitWhitespace from '../libs/splitWhitespace';
+const $w = splitWhitespace;
+import * as statHelpers from '../statHelpers';
+
+module.exports = function statsComputed (user) {
+  let thisArr = $w('stats stats.buffs items.gear.equipped.weapon items.gear.equipped.armor items.gear.equipped.head items.gear.equipped.shield');
```

Comment:
Updated.

2016-04-11 15:45 GMT-07:00 Blade Barringer <notifications@github.com>:

> In common/script/fns/statsComputed.js
> <https://github.com/HabitRPG/habitrpg/pull/7067#discussion_r59294744>:
>
> > @@ -0,0 +1,28 @@
> > +import _ from 'lodash';
> > +import content from '../content/index';
> > +import splitWhitespace from '../libs/splitWhitespace';
> > +const $w = splitWhitespace;
> > +import * as statHelpers from '../statHelpers';
> > +
> > +module.exports = function statsComputed (user) {
> > +  let thisArr = $w('stats stats.buffs items.gear.equipped.weapon items.gear.equipped.armor items.gear.equipped.head items.gear.equipped.shield');
>
> Agreed.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/HabitRPG/habitrpg/pull/7067/files/586749613f859d411458dc96cc9b07c64cfb1748#r59294744>
>



-- 
Victor Pudeyev
piousbox@gmail.com